sway-core/src/asm_generation/fuel/register_allocator.rs

```diff
@@ -777,6 +777,7 @@ fn assign_registers(
             .neighbors_undirected(node)
             .map(|neighbor| interference_graph[neighbor].clone())
             .collect();
+        // println!("assign_registers: {:?}", neighbors);
```

Comment:
Do we want to keep this commented-out line?

Reply:
Yeah, no we don't. Removed.